website/docs/d/pi_network_interface.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_pi_network_interface"
```

Comment:
```suggestion
page_title: "IBM : pi_network_interface"
```

Reply:
@yussufsh Shouldn't this be same as the title? `# ibm_pi_network_interface`
That is what other services do from taking a look at most of them.

Reply:
We have use with `ibm_` prefix at some places. But I guess it does not matter, we need to set a standard at least of Power resources.
Check if we can use https://github.com/hashicorp/terraform-plugin-docs for generating the docs using templates. This is what the default template looks like: https://github.com/hashicorp/terraform-plugin-docs/blob/12a6bfdd85fcc55f755be3e7871b00b24ccd8e15/internal/provider/template.go#L217-L244
cc @hkantare